src/libraries/System.Numerics.Vectors/tests/Matrix4x4Tests.cs

```diff
@@ -819,11 +819,57 @@ public void Matrix4x4CreateReflectionTest01()
                     Vector3 v = point - pp;
                     float d = Vector3.Dot(v, plane.Normal);
                     Vector3 vp = point - 2.0f * d * plane.Normal;
-                    Assert.True(MathHelper.Equal(rp, vp), "Matrix4x4.Reflection did not provide expected value.");
+
+                    AssertExtensions.Equal(vp.M11, rp.M11, 0.0f);
```

Comment:
`vp` and `rp` are Vector3, not Matrix4x4

Reply:
You are right. The `Matrix4x4` in the assertion message misled me.